AmplifyPlugins/Predictions/AWSPredictionsPluginIntegrationTests/AWSPredictionsPluginTestBase.swift

```diff
@@ -6,97 +6,43 @@
 //
 
 import XCTest
-import AWSMobileClient
-@testable import Amplify
+import AmplifyPlugins
 import AWSPredictionsPlugin
 
+@testable import Amplify
+
 class AWSPredictionsPluginTestBase: XCTestCase {
 
-    let region: JSONValue = "us-west-2"
     let networkTimeout = TimeInterval(180) // 180 seconds to wait before network timeouts
 
     override func setUp() {
-        setupMobileClient()
-        setupAmplify()
-    }
+        super.setUp()
 
-    override func tearDown() {
-        sleep(1)
-        print("Amplify reset")
-        Amplify.reset()
-    }
+        continueAfterFailure = false
 
-    private func setupAmplify() {
-        // Set up Amplify predictions configuration
-        let predictionsConfig = PredictionsCategoryConfiguration(
-            plugins: [
-                "awsPredictionsPlugin": [
-                    "defaultRegion": region,
-                    "identify": [
-                        "identifyEntities": [
-                        "maxFaces": 50,
-                        "collectionId": "", //no collectionid
-                        "region": region
-                        ]
-                    ],
-                    "convert": [
-                        "transcription": [
-                            "language": "en",
-                            "region": region,
-                            "defaultNetworkPolicy": "auto"
-                        ]
-                    ]
-                ]
-            ]
-        )
+        Amplify.reset()
 
-        let amplifyConfig = AmplifyConfiguration(predictions: predictionsConfig)
```

Comment:
According to Jithin, there is no specific reason why not using one over the other. Since the other categories use file way of configuring, then I would also adopt that way and it's more convenient.